dark_evaluation.py

```diff
@@ -27,45 +47,38 @@
     digicam_geometry = geometry.generate_geometry_from_camera(camera=digicam)
 
     # Define the event stream
-    data_stream = event_stream(file_list=file_list, expert_mode=True, camera_geometry=digicam_geometry)
-    data_stream = filter.set_patches_to_zero(data_stream, unwanted_patch=unwanted_patch)
+    data_stream = event_stream(
+        file_list=files,
+        expert_mode=True,
+        camera_geometry=digicam_geometry,
+        camera=digicam
+    )
+    data_stream = filter.set_patches_to_zero(
+        data_stream,
+        unwanted_patch=unwanted_patch)
+    data_stream = r0.fill_trigger_input_7(data_stream)
     # Fill the flags (to be replaced by Digicam)
     data_stream = filter.filter_event_types(data_stream, flags=[8])
 
-    data_stream = save_bias_curve(data_stream,
-                                  thresholds=thresholds,
-                                  camera=digicam,
-                                  blinding=blinding,
-                                  output_filename=directory + trigger_filename,
-                                  unwanted_cluster=unwanted_cluster)
-
-    data_stream = save_dark(data_stream, directory + dark_filename)
-
-    for i, data in enumerate(data_stream):
-
-        print(i)
-
-    data_dark = np.load(directory + dark_filename)
-    data_rate = np.load(directory + trigger_filename)
-
-    plt.figure()
-    plt.hist(data_dark['baseline'], bins='auto')
-    plt.xlabel('dark baseline [LSB]')
-    plt.ylabel('count')
+    data_stream = save_bias_curve(
+        data_stream,
+        thresholds=thresholds,
+        blinding=blinding,
+        output_filename=dark_trigger_file_path,
+        unwanted_cluster=unwanted_cluster
+    )
 
-    plt.figure()
-    plt.hist(data_dark['standard_deviation'], bins='auto')
-    plt.xlabel('dark std [LSB]')
-    plt.ylabel('count')
+    data_stream = save_dark(data_stream, dark_file_path)
 
-    fig = plt.figure()
-    axis = fig.add_subplot(111)
-    axis.errorbar(data_rate['threshold'], data_rate['rate'] * 1E9, yerr=data_rate['rate_error'] * 1E9, label='Blinding : {}'.format(blinding))
-    axis.set_ylabel('rate [Hz]')
-    axis.set_xlabel('threshold [LSB]')
-    axis.set_yscale('log')
-    axis.legend(loc='best')
-    plt.show()
+    for _ in tqdm(data_stream):
+        pass
 
 
-    ## Filter the events for display
+    # Filter the events for display
+if __name__ == "__main__":
+    args = docopt(__doc__)
+    print(args)
+    main(
+        output_directory=args['<output_directory>'],
+        files=args['<files>']
```

Comment:
can we have default values from one of the test files ?

Reply:
erm .. not with docopt ... i.e. it would be very ugly with docopt ... let me show you.

Reply:
But where do you want to write the output? the current working directory, so the dot `.`?

Reply:
Ah .. there is another "problem". I implemented these arguments as *mandatory* arguments. So it is as if I said:

This program cannot possibly guess any useful default input. It cannot guess where you want the output to be and it cannot guess where you have your SST1M files lying around. 

Like `mv` cannot guess what you want to move where. And `tar` cannot guess what you wanna compress.

Can I ask you, why you want to say to the user: "Hey, in case you have no idea what to do: Just use one of the example files ... it's fine".

Reply:
yes current directory seems ok ? ok should we have `null/` folder in the project?... maybe not.
I would think it is good if we use those template files such that we can launch easily "demos".

Also maybe adding python `dark_evaluation.py a_template_file` command to travis could help to check if it at least runs?

Reply:
Okay have a look, now these are not mandatory arguments anymore but options, with default values